packages/aws/data_stream/firewall_logs/manifest.yml

```
@@ -216,7 +216,7 @@ streams:
         required: false
         default: beginning
         show_user: true
-        description: Allows user to specify if this input should read log files from the beginning or from the end.
+        description: Specify if the input should start reading logs from from the `beginning` (oldest log entry), `end` (newest log entry), or `lastSync` (last successful read timestamp if input ran before).
```

```suggestion
        description: Specify if the input should start reading logs from the `beginning` (oldest log entry), `end` (newest log entry), or `lastSync` (last successful read timestamp if input ran before).
```
(throughout)

Also consider `s/if/whether/`.

---

Thanks will fix this